configuration/cardano/mainnet-config.json

```diff
@@ -1,6 +1,8 @@
 {
   "AlonzoGenesisFile": "mainnet-alonzo-genesis.json",
   "AlonzoGenesisHash": "7e94a15f55d1e82d10f09203fa1d40f8eede58fd8066542cf6566008068ed874",
+  "ApplicationName": "cardano-sl",
```

Comment:
Why it's needed?

Reply:
Did that not get removed?

Reply:
I think a rebase or cherry-pick picked up more than I intended.

Reply:
Fixed now.